src/benchq/mlflow/data_logging.py

```diff
@@ -0,0 +1,68 @@
+################################################################################
+# © Copyright 2023 Zapata Computing Inc.
+################################################################################
+""" Tools for using mlflow to log benchq data """
+from dataclasses import asdict
+from numbers import Number
+from typing import Any, Dict, Optional
+
+import mlflow  # type: ignore
+
+from ..data_structures import (
+    AlgorithmImplementation,
+    BasicArchitectureModel,
+    DecoderModel,
+    ResourceInfo,
+)
+
+
+def log_input_objects_to_mlflow(
+    algorithm_description: AlgorithmImplementation,
+    algorithm_name: str,
+    hardware_model: BasicArchitectureModel,
+    decoder_model: Optional[DecoderModel] = None,
+):
```

Comment:
Add `-> None` to indicate return value type

Reply:
fixed in ba87b5c7cf219232747b7e560dae8ec237e610c5